app/utils/discogs.ts

```diff
@@ -1,26 +1,29 @@
 import { z } from "zod";
-import { SearchResponseSchema } from "./discogs.schemas";
+import { Album, SearchResponseSchema } from "./discogs.schemas";
 
 const discogs = {
-  search: async (search: string): Promise <z.infer<typeof SearchResponseSchema>> => {
+  search: async (
+    search: string
+  ): Promise<z.infer<typeof SearchResponseSchema>> => {
     // define url for GET API call
     const searchParams = new URLSearchParams({
       q: search,
       type: "release",
       token: process.env.DISCOGS_API_KEY!,
+      country: "US",
+      format: "Vinyl",
     });
+
     const url = "https://api.discogs.com/database/search?" + searchParams;
-  
-    console.log("url: ", url);
-  
+
     // GET call to Discogs API
     const result = await fetch(url);
     const data = await result.json();
-  
-    // schema check using Zod
-    SearchResponseSchema.parse(data);
+
+      // schema check using Zod
+      SearchResponseSchema.parse(data);
```

Comment:
the return value from this is what you want to return since Zod won't include fields that you don't specify in your schema